bin/load-runner.js

```diff
@@ -98,6 +108,10 @@ var outputDir = null;
 var saveOutput = false;
 var writeInProgess = 0;
 
+//initialize random number generator
+const seed = parseInt(args.seed);
+const rand = isNaN(seed) ? lcg() : lcg(seed);
```

Comment:
There's still the possibility for any number less than 1 here, which would cause a TypeError I believe. It's probably fine since it will fail early anyway.